client/sender.go

```diff
@@ -85,3 +86,46 @@ func newSender(u *url.URL, ctx *base.Context, retryOptions retry.Options, stoppe
 	}
 	return f(u, ctx, retryOptions, stopper)
 }
+
+// SendWrapped is a convenience function which wraps the request in a batch,
+// sends it via the provided Sender, and returns the unwrapped response
+// or an error. It's valid to pass a `nil` context; context.TODO() is used
```

Comment:
We should use `context.Background()` in this case. The point of `context.TODO()` is that it's something that you can grep for with the aim of removing. Most functions that accept a `context.Context` do not accept nil and expect the caller to choose either `context.TODO()` (if they expect to change to another context in the future) or `context.Background()` (if this is a permanent state). Burying the call to `context.TODO()` in a library function defeats the greppability so it might as well be `Background` instead (the two are exactly equivalent).